benchclients/python/benchclients/logging.py

```diff
@@ -1,9 +1,12 @@
 import logging
 
+log = logging.getLogger(__name__)
+
 logging.basicConfig(
-    format="%(levelname)s [%(asctime)s] %(message)s", level=logging.INFO
```

Comment:
compared to the 'old' format this adds fractions of a second with millisecond resolution: `%(msecs)03d`